src/components/toolbar/OrganizationToolBar.tsx

```diff
@@ -0,0 +1,79 @@
+import {Toolbar, ToolbarContent, ToolbarItem} from '@patternfly/react-core';
+import {DropdownCheckbox} from './DropdownCheckbox';
+import {DropdownFilter} from './DropdownFilter';
+import {FilterBar} from './FilterBar';
+import {ToolbarButton} from './ToolbarButton';
+import {Kebab} from './Kebab';
+import {ToolbarPagination} from './Pagination';
+import {filterOrgState} from 'src/atoms/UserState';
+import {useRecoilState} from 'recoil';
+
+export function OrganizationToolBar(props: OrganizationToolBarProps) {
+  const [filterOrg, setOrgFilter] = useRecoilState(filterOrgState);
+  const filterOrgs = (value: string) => {
+    setOrgFilter(value);
+  };
+
+  return (
+    <Toolbar>
+      <ToolbarContent>
+        <DropdownCheckbox
+          selectedItems={props.selectedOrganization}
+          deSelectAll={props.setSelectedOrganization}
+          allItemsList={props.organizationsList}
+          itemsPerPageList={props.paginatedOrganizationsList}
+          onItemSelect={props.onSelectOrganization}
+        />
+        <DropdownFilter />
+        <FilterBar
+          placeholderText="Filter by name"
+          value={filterOrg}
+          onChange={filterOrgs}
+        />
+        <ToolbarButton
+          buttonValue="Create Organization"
+          Modal={props.createOrgModal}
+          isModalOpen={props.isOrganizationModalOpen}
+          setModalOpen={props.setOrganizationModalOpen}
+        />
+        <ToolbarItem>
+          {props.selectedOrganization.length !== 0 ? (
```

Comment:
can `selectedOrganization` be `null` anytime? if so this can cause an exception

Reply:
+1, could do [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) - `props.selectedOrganization?.length !== 0`

Reply:
Didn't know about optional chaining. That's a wonderful syntax.